src/hooks/useIsProfileValidated.ts

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react'
+
+import useAsyncMemo from 'decentraland-gatsby/dist/hooks/useAsyncMemo'
+
+import { Governance } from '../clients/Governance'
+
+function useIsProfileValidated(address: string | null) {
+  const [isLoading, setIsLoading] = useState(true)
+  const [isProfileValidated] = useAsyncMemo(
+    async () => {
+      setIsLoading(true)
+      let isValidated = false
+      if (address) {
```

Comment:
Can't we assume address exists here if you are calling this with truthy deps and the address as a dep?